package.xml

```diff
@@ -14,6 +14,7 @@
 
   <depend>boost</depend>
   <depend>builtin_interfaces</depend>
+  <depend>libusb-1.0-dev</depend>
```

Comment:
Are you sure this dependency works?  The Dockerfile has this:

libusb-1.0.0-dev